javascript/ql/src/Security/CWE-352/MissingCsrfMiddleware.qhelp

```diff
@@ -58,5 +58,6 @@
 
 <references>
 	<li>OWASP: <a href="https://www.owasp.org/index.php/Cross-Site_Request_Forgery_(CSRF)">Cross-Site Request Forgery (CSRF)</a></li>
+	<li>< ahref ="https://www.npmjs.com/package/csurf"</a></li>
```

Comment:
Adding a link to `csurf` sound like a fine idea, but your suggestion is not valid XML. 

```suggestion
	<li>NPM: <a href="https://www.npmjs.com/package/csurf">csurf</a></li>
```